cookbook/integrations/kubernetes/k8s_spark/pyspark_pi.py

```diff
@@ -41,6 +41,34 @@ def hello_spark(partitions: int) -> float:
            # Regular Pypsark code
            ...
 
+# To run a Spark job on Databricks platform, just add Databricks config to the task config. Databricks Config is is the same as the databricks job request. Refer to `Databricks job request <https://docs.databricks.com/dev-tools/api/2.0/jobs.html#request-structure>`_
+
+   .. code-block:: python
+
+       @task(
+           task_config=Spark(
+               # this configuration is applied to the spark cluster
+               spark_conf={
+                   "spark.driver.memory": "1000M",
+                   "spark.executor.instances": "2",
+                   "spark.driver.cores": "1",
+               },
+               databricks_conf={
+                   "run_name": "flytekit databricks plugin example",
+                   "new_cluster": {
+                       "spark_version": "11.0.x-scala2.12",
+                       "node_type_id": "r3.xlarge",
+                       "aws_attributes": {
+                           "availability": "ON_DEMAND",
+                           "instance_profile_arn": "arn:aws:iam::590375263360:instance-profile/databricks-s3-role",
```

Comment:
can you change this to some random numbers?

Reply:
is there any permissioning stuff users need to be aware of?  what flyte buckets does this role need access to?

Reply:
admin should create this instance profile for used before running the task, and this role should have access to the bucket where we save task's input and output